pkg/artifactory/datasource/user/datasource_artifactory_user_test.go

```diff
@@ -7,18 +7,10 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 	"github.com/jfrog/terraform-provider-artifactory/v7/pkg/acctest"
-	"github.com/jfrog/terraform-provider-artifactory/v7/pkg/artifactory/resource/user"
 	"github.com/jfrog/terraform-provider-shared/test"
 	"github.com/jfrog/terraform-provider-shared/util"
 )
 
-func deleteUser(t *testing.T, name string) error {
```

Comment:
Why is this deleted?

Reply:
Not used at all. This private func has been replaced by a public one https://github.com/jfrog/terraform-provider-artifactory/blob/master/pkg/acctest/test.go#L282